src/app/screens/Nostr/ConfirmSignMessage.tsx

```diff
@@ -137,10 +147,10 @@ function ConfirmSignMessage() {
                   }),
                 })}
               />
-              <ConfirmOrCancel
+              <SignOrDeny
                 disabled={loading}
                 loading={loading}
-                onCancel={reject}
+                onDeny={reject}
```

Comment:
I think there is a difference between closing the window and actively clicking deny. 

Closing the window = Cancel the request
Deny = Cancel the request + add a block permission

Does that make sense @stackingsaunter ?

Reply:
make sense